documentation/API-design-guidelines.md

```diff
@@ -1244,8 +1244,8 @@ In order to ease developer adoption, the pattern for Resource-based event subscr
 | ----- |	-----  |	 -----  | 
 | POST | `/subscriptions` |  Operation to request an event subscription.     |
```

Comment:
Add sentence (proposal): 
As the subscription could be created synchronously or asynchronously both status code 201 and 202 must be described in the yaml.